<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>626</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #A4A8A6;</string>
  </property>
  <widget class="QLabel" name="label_error">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>640</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #FF0000;
font-size: 12px;
font-weight: bold;
</string>
   </property>
   <property name="text">
    <string>LABEL ERROR</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="combobox_subj">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>590</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFF5B2;
border-radius:5px;
border: 1px groove;
border-color: #FFCF38;
font-size: 15px;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>510</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font-size : 15px;
background-color : #05BF11;
color: &quot;white&quot;;
font-weight: bold;
border: 1px solid;
border-radius: 4px;
border-color: #95FF93;
}

 QPushButton:hover
{
background-color : green;
}</string>
   </property>
   <property name="text">
    <string>A D D</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>370</y>
     <width>641</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
font-weight: bold;
font-size: 18px;
border: 1px solid;
background-color: #06A310;</string>
   </property>
   <property name="text">
    <string>R E V I E W E R</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_load">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>680</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font-size : 15px;
background-color : #05BF11;
color: &quot;white&quot;;
font-weight: bold;
border: 1px solid;
border-radius: 4px;
border-color: #95FF93;
}

 QPushButton:hover
{
background-color : green;
}</string>
   </property>
   <property name="text">
    <string>Load Subject</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>680</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
font-size : 15px;
background-color : #05BF11;
color: &quot;white&quot;;
font-weight: bold;
border: 1px solid;
border-radius: 4px;
border-color: #95FF93;
}

 QPushButton:hover
{
background-color : green;
}</string>
   </property>
   <property name="text">
    <string>Delete Subject</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_subj">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>470</y>
     <width>521</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:#FFF5B2;
border-radius:5px;
border: 1px groove;
border-color: #FFCF38;
font-size: 15px;</string>
   </property>
   <property name="placeholderText">
    <string> Enter a New Subject here then click &quot;ADD&quot; </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_subjects">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>560</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size : 15px;
color: #0048FF;

font-weight: bold;
border-color: #95FF93;</string>
   </property>
   <property name="text">
    <string>  CHOOSE SUBJECT</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>-20</y>
     <width>401</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #e9e9e9;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>images/reader-310398.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>391</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #e9e9e9;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_add">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>440</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size : 15px;
color: #0048FF;

font-weight: bold;
border-color: #95FF93;</string>
   </property>
   <property name="text">
    <string>  ADD SUBJECT</string>
   </property>
  </widget>
  <zorder>label_bg</zorder>
  <zorder>label</zorder>
  <zorder>label_error</zorder>
  <zorder>combobox_subj</zorder>
  <zorder>btn_add</zorder>
  <zorder>label_title</zorder>
  <zorder>btn_load</zorder>
  <zorder>btn_delete</zorder>
  <zorder>txt_subj</zorder>
  <zorder>label_subjects</zorder>
  <zorder>label_add</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
